qutebrowser/browser/hints.py

```diff
@@ -591,6 +591,18 @@ def _start_cb(self, elems):
             message.error("No elements found.")
             return
 
+        # Because _start_cb is called asynchronously, it's possible that the
+        # user switched to another tab or closed the tab/window. In that case
+        # we should not start hinting.
+        tabbed_browser = objreg.get('tabbed-browser', default=None,
+                scope='window', window=self._win_id)
+        if tabbed_browser is None:
+            return
+        tab = tabbed_browser.widget.currentWidget()
+        if tab.tab_id != self._tab_id:
```

Comment:
Is there any case that tab is None?

Reply:
Or the `tabbed_browser` object above. I think at this time, the tab should not be closed (because otherwise `find_css` would raise an error), so `tabbed_browser` cannot be None.